src/XMonad/Operations.hs

```diff
@@ -481,6 +481,15 @@ screenWorkspace sc = withWindowSet $ return . W.lookupWorkspace sc
 withFocused :: (Window -> X ()) -> X ()
 withFocused f = withWindowSet $ \w -> whenJust (W.peek w) f
 
+-- | Apply an 'X' operation to all unfocused windows, if there are any.
+withUnfocused :: (Window -> X ()) -> X ()
+withUnfocused f = withWindowSet $ \ws ->
+    case W.peek ws of
+        Just w ->
+            let unfocusedWindows = filter (/= w) $ W.index ws
+            in mapM_ f unfocusedWindows
+        Nothing -> pure ()
```

Comment:
It would probably be a good idea to use X.Core's `whenJust` here, just like `withFocused` does

Reply:
updated!